src/ddceler/DDcelerRunAction.cc

```diff
@@ -0,0 +1,39 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file ddceler/DDcelerRunAction.cc
+//---------------------------------------------------------------------------//
+#include "DDcelerRunAction.hh"
+
+#include <CeleritasG4.hh>
+#include <DDG4/Factories.h>
+
+using TMI = celeritas::TrackingManagerIntegration;
+
+namespace dd4hep
+{
+namespace sim
+{
+
+//---------------------------------------------------------------------------//
+
+void DDcelerRunAction::begin(G4Run const* run)
+{
+    this->info("Begin of run");
```

Comment:
These are probably superfluous since the TMI instance also logs similar. If we should be redirecting logging through DD4HEP we can do that as well.